src/test/java/pl/edu/mimuw/BigIntTest.java

```diff
@@ -29,17 +32,34 @@ void testBigAdd() {
   }
 
   @Test
-  void testSmallTimes() {
-    throw new IllegalStateException("TODO task 5: write test for multiplication of small numbers");
+  void testSmallTimes()
```

Comment:
ok, tylko bym wyrzucił te 20 do zmiennych na początku metody, żeby było jasne ze to granice testowania